app/code/GorbanSv/CustomWidget/view/frontend/templates/widget/banner.phtml

```diff
@@ -0,0 +1,16 @@
+<?php
+/* @var $block \GorbanSv\CustomWidget\Block\Widget\Banner */
+?>
+
+<?php if ($block->getData('button_title')): ?>
+    <button class="button-cms-page">
+        <a href="<?php echo $block->getUrl('geekhub-cms') ?>"><?php echo $block->getData('button_title') ?></a>
+    </button>
+<?php endif; ?>
+
+<?php if ($block->getData('cms_block')): ?>
```

Comment:
Move check and rendering functional in block class, and use set/get magic functions like $block->getCmsBlock()